crates/aptos/src/move_tool/mod.rs

```diff
@@ -222,6 +226,47 @@ impl CliCommand<&'static str> for TestPackage {
     }
 }
 
+/// Prove the Move package at the package path
+#[derive(Parser)]
+pub struct ProvePackage {
+    #[clap(flatten)]
+    move_options: MovePackageDir,
+
+    /// A filter string to determine which unit tests to run
+    #[clap(long)]
+    pub filter: Option<String>,
+}
+
+#[async_trait]
+impl CliCommand<&'static str> for ProvePackage {
+    fn command_name(&self) -> &'static str {
+        "ProvePackage"
+    }
+
+    async fn execute(self) -> CliTypedResult<&'static str> {
+        let config = BuildConfig {
+            additional_named_addresses: self.move_options.named_addresses(),
+            test_mode: true,
+            install_dir: self.move_options.output_dir.clone(),
+            ..Default::default()
+        };
+        let result = task::spawn_blocking(move || {
+            move_cli::base::prove::run_move_prover(
+                config,
+                self.move_options.package_dir.as_path(),
+                &self.filter,
+                false,
+                move_prover::cli::Options::default(),
+            )
+        })
+        .await.unwrap();
```

Comment:
Unwraps are an anti-pattern in the Aptos CLI, please make a mapping to CliError :)

Even better would be to provide a useful error message if it doesn't have one.

Reply:
Got it. Done.